src/test/java/com/xceptance/neodymium/tests/DriverArgumentsTest.java

```diff
@@ -0,0 +1,48 @@
+package com.xceptance.neodymium.tests;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.codeborne.selenide.Selenide;
+import com.xceptance.neodymium.NeodymiumRunner;
+import com.xceptance.neodymium.module.statement.browser.multibrowser.Browser;
+
+@RunWith(NeodymiumRunner.class)
+@Browser("Chrome_headless")
+@Browser("FF_headless")
+public class DriverArgumentsTest extends NeodymiumTest
```

Comment:
please add a test for an empty config string as well.

Reply:
implicity covered with other tests, adding tests for invalid values will test browser behavior but not our code